core/dropdowndiv.js

```diff
@@ -202,14 +190,19 @@ Blockly.DropDownDiv.show = function(owner, primaryX, primaryY, secondaryX, secon
     metrics.arrowX + 'px,' + metrics.arrowY + 'px) rotate(45deg)';
   Blockly.DropDownDiv.arrow_.setAttribute('class',
     metrics.arrowAtTop ? 'blocklyDropDownArrow arrowTop' : 'blocklyDropDownArrow arrowBottom');
-  // First apply initial translation
-  div.style.transform = 'translate(' + metrics.finalX + 'px,' + metrics.finalY + 'px)';
-  // Save for animate out
-  Blockly.DropDownDiv.hideAnimationX_ = metrics.initialX;
-  Blockly.DropDownDiv.hideAnimationY_ = metrics.initialY;
-  // Show the div
+  // First apply initial translation.
+  // We put this in the `left` and `top` properties to avoid colliding with `translate`.
```

Comment:
Longer comment on what you mean by collision--we might not remember why we did this down the line.

Here's the start of a comment.  Feel free to change it completely.

// When we change 'translate' multiple times in close succession, chrome may choose to wait and apply them all at once.  Since we want the first translation to be immediate and the second to be animated, we saw problems where both translations would be applied after animation was turned on, making the dropdown appear to fly in from (0, 0).  This is a workaround.